Chat/ClientApp/src/components/ChatHeader.tsx

```diff
@@ -54,7 +56,10 @@ export default (props: ChatHeaderDispatchProps & ChatHeaderProps & ChatHeaderPan
   }, [topic, generateHeaderMessage]);
 
   return (
-    <Stack className={chatHeaderContainerStyle} horizontal={true} horizontalAlign="space-between">
+      <Stack className={chatHeaderContainerStyle} horizontal={true} horizontalAlign="space-between">
+      <Stack.Item grow={1} className={feedbackContainer}>
+        <FeedbackButton />
```

Comment:
Indent looks a little bit off, please fix the indent for the top one 'stack'

Reply:
will fix that and push the update out.

Reply:
Fixed the indentation issue.